src/Items/ItemFishingRod.h

```diff
@@ -236,9 +239,11 @@ class cItemFishingRodHandler :
 				{
 					return true;
 				}
-				Vector3d FloaterPos = FloaterInfo.GetPos();
+				Vector3d FloaterPos = FloaterInfo.GetBitePos();
+				const float FISH_SPEED_MULT_XZ = 2.7f;
+				const float FISH_SPEED_MULT_Y = 2.0f;
```

Comment:
Do these numbers work correctly even while the player is closer or further away from the floater?